### DISCUSSION TITLE
More Birthday/Anninversary Reminders

### DISCUSSION THREAD

**Discussion by trymeouteh** [FossifyOrg/General-Discussion#421] [Ideas]
Please allow for more than 3 birthday/anniversary reminders. Please allow for 8 reminders.

**Comment by Aga-C** [comment]
You can create a new Feature Request for that in `Issues` tab. Fill out the template describing how you'd like it to work.

Also, there is no difference in usual events and birthday/anniversaries, so I'm closing the second discussion (https://github.com/FossifyOrg/Calendar/discussions/134). You can write about both cases in your feature request, but don't create two separate.
